### DISCUSSION TITLE
v1.7.4

### DISCUSSION THREAD

**Discussion by 0xJacky** [0xJacky/nginx-ui#71] [Announcements]
# Features
1. Inspect configurations.

For more change log about v.1.7.0 please check [v1.7.0 Release Note](https://github.com/0xJacky/nginx-ui/releases/tag/v1.7.0-patch).

<hr /><em>This discussion was created from the release <a href='https://github.com/0xJacky/nginx-ui/releases/tag/v1.7.4'>v1.7.4</a>.</em>

**Comment by mohamad-supangat** [comment]
great feature, tanks for release
